crates/services/p2p/src/request_response/messages.rs

```diff
@@ -76,6 +83,7 @@ pub enum NetworkResponse {
 pub enum OutboundResponse {
     Block(Option<Arc<SealedBlock>>),
     SealedHeader(Option<Arc<SealedBlockHeader>>),
+    SealedHeadersRangeInclusive(Vec<SealedBlockHeader>),
```

Comment:
Just curious:
- Why doesn't this one follow the same pattern of `Option<Arc<..>>` like the others?
- Is there a particular reason you went with the name `SealedHeadersRangeInclusive` instead of just `SealedHeaders`, similar to the other variants?

Reply:
> Why doesn't this one follow the same pattern of Option<Arc<..>> like the others?
I consider an empty `Vec` to be equivalent to a `None`. IF we were considering doing `Vec<Option<...>>` I'd still have to do a check on the ordering of the response, so it also didn't seem to give any meaning at this level. I discussed this a bit with @xgreenx before and I'm don't think he objected to my reasoning. Still open to changing if I'm wrong.

WRT the `Arc` I left it out expecting to get some `Sync`/`Send` error from the compiler if I needed it. Never got an error. Probably superfluous in the other cases.

> Is there a particular reason you went with the name SealedHeadersRangeInclusive instead of just SealedHeaders, similar to the other variants?
That's just left behind from when I wasn't using `range` and I've changed the name everywhere else. I'll fix that. Thanks!